src/plugins/home/public/application/components/guided_onboarding/use_case_card.tsx

```diff
@@ -113,7 +101,17 @@ export interface UseCaseProps {
 }
 
 export const UseCaseCard = ({ useCase }: UseCaseProps) => {
-  const { application, trackUiMetric } = getServices();
+  const { application, trackUiMetric, uiSettings, http } = getServices();
+
+  const isDarkTheme = uiSettings.get('theme:darkMode');
+
+  const getImageUrl = (imageName: UseCase) => {
+    const imagePath = isDarkTheme
+      ? `/plugins/home/assets/solution_logos/${imageName}_dark.png`
+      : `/plugins/home/assets/solution_logos/${imageName}.png`;
```

Comment:
nit: this would be greatly simplified as:

```
const imagePath = `/plugins/home/assets/solution_logos/${imageName}${isDarkTheme ? '_dark' : ''}.png`
```